docs/BasicCommunication/GetAppProperties/assets/GetAppProperties.txt

```diff
@@ -0,0 +1,34 @@
+title GetAppProperties 
+
+participant Local Web App
+participant SDL
+participant HMI
+
+==<align:center>Pre-conditions: 1. SDL and HMI are started\n 2. application's properties were set via BC.SetAppProperties from HMI==  
+
+opt GetAppProperties - Happy path
+HMI -> SDL: BC.GetAppProperties(policyAppID:"123")
+SDL-->HMI: BC.GetAppProperties(policyAppID:"123", SUCCESS)
+end 
+
+opt GetAppProperties - request without `policyAppID`
+HMI -> SDL:BC.GetAppProperties()
+SDL-->HMI: BC.GetAppProperties(all applications properties, SUCCESS)
+end
+
+opt GetAppProperties - request with wrong `policyAppID`
+HMI -> SDL: BC.GetAppProperties(policyAppID:"1 \2qwer'ty3")
+SDL-->HMI: BC.GetAppProperties(DATA_NOT_AVAILABLE)
+end
+
+opt GetAppProperties - request with unknown parameter
+HMI -> SDL: BC.GetAppProperties(unknown parameter:unknown parameter, policyAppID:"123",enabled:true)
+SDL ->SDL: Cut off the unknown parameter
+
+note over SDL:Process the request as assigned
+end
+
+opt GetAppProperties - request with invalid parameters
+HMI -> SDL: BC.GetAppProperties(policyAppID:123)
```

Comment:
took me a second to realize this is an integer and not the same string as the first case, maybe we could make it more obvious with `0x` hex prefix or maybe just using a different number than 123?

Reply:
please, see updates in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/187/commits/4ebba070bd78b95c3a00721cc2bcb639a6dfb422